openpype/settings/entities/schemas/projects_schema/schemas/schema_nuke_imageio.json

```diff
@@ -6,8 +6,14 @@
     "is_group": true,
     "children": [
         {
-            "type": "label",
-            "label": "<b style='color:red'>'Custom OCIO config path'</b> has deprecated. <br> If you need to set custom config, just enable and add path into 'OCIO config'. <br>Anatomy keys are supported.</i>."
+            "type": "boolean",
+            "key": "activate_host_color_management",
+            "label": "Enable Color Management in host"
+        },
+        {
+            "type": "boolean",
+            "key": "use_ocio_config",
+            "label": "Use OCIO config file in host"
```

Comment:
I don't understand the difference between these two options. Aren't these two mutually exclusive?

What happens if:
- `use_ocio_config` is true but `activate_host_color_management` is false - how is it using a ocio config if we consider the host not color managed?
- `use_ocio_config` is false but `activate_host_color_management` is true - what is it doing then?

Aren't these the exact same setting?

Reply:
> use_ocio_config is true but activate_host_color_management is false - how is it using a ocio config if we consider the host not color managed?

This is something I am not able to make more interactive in the current Settigs gui. We do not have connected signals so once attribut would effect the other to be hidden or disabled. But to answer your question if host colormanagement is deactivated and use of config is activated the priority is on host management and use config is secondary. 

> use_ocio_config is false but activate_host_color_management is true - what is it doing then?

This is more itneresting. Since some hosts are having available own build in ocio configs (nuke, maya, etc) we can either have activated just the colormanagedment and take advantage of file_rules and colorspaceData distribution or we can activate `use_ocio_config` and push our ocio convig files into the host wheather it is global or host overrides. 

Notice that some hosts are not havin this attribute available and activated can be only host's colorspace management. This is here for hosts which are either not colormanaged internally at all or have only its own configs. This way we can wrap it during publishing with out colormanagement and with use of File rules to match colorspace names found in our configs.

Reply:
It's funny that even with the explanation I don't understand. @fabiaserra any chance you do and can dumb it down for me?

Reply:
I understood the first part yeah, that unfortunately the settings logic doesn't have UI conditionals that allows us to disable/hide options depending on some other value, so while you are right that those options are mutually exclusive... we have no alternative but to give priority to one setting over the other. On this case, I understand that `activate_host_color_management` takes priority and so no matter what you have on `use_ocio_config`, the setting is ignored as long as it's `False`?

On the contrary, if `activate_host_color_management` is `True`... some hosts might use the internal (in my opinion legacy) color management settings to do some color management without an OCIO file.


Personally I would rather not support legacy color management workflows and try enforce new adopters of OP to just use OCIO on all hosts that support it, but I don't know if that's too much of an ask

Reply:
Thanks @fabiaserra for the interpretation. Yes it is correct :heart_eyes:

> Personally I would rather not support legacy color management workflows and try enforce new adopters of OP to just use OCIO on all hosts that support it, but I don't know if that's too much of an ask

Yes I see your point and say yes you can configure your studio this way, but someone might want to use it differently and this should be universal enoughh to give an option to them.

Farther details: https://github.com/ynput/OpenPype/pull/4700#discussion_r1173505722